### ISSUE TITLE
[QUESTION] Do you override the layer normalization in LnMlpPolicy when defining the policy_kwargs?

### ISSUE THREAD

**Issue by jandevries123** [hill-a/stable-baselines#939]
When using DQN and you choose the LnMlpPolicy, when defining the policy kwargs as eg `policy_kwargs=dict(act_fun=tf.nn.leaky_relu, layers=[256, 256])`  Do you override the normalization of the layers or does it changes the layers and it keeps the normalization? Also which layer normalization methods are applied when using the LnMlpPolicy, i couldn't find it? 

Reference:
https://stable-baselines.readthedocs.io/en/master/modules/dqn.html#stable_baselines.deepq.LnMlpPolicy

thanks in advance

**Comment by Miffyli** [comment]
Answers are [here in the code](https://github.com/hill-a/stable-baselines/blob/master/stable_baselines/deepq/policies.py#L113). The normalization comes before activation function,. Layer normalization is applied after each layer in "Ln" policies.

**Comment by jandevries123** [comment]
please delete
